### PR TITLE
Removed unnecessary product files and vimeo dependencies

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -27,10 +27,10 @@ gem 'git', '~> 1.2.5'
 gem 'oily_png', '~> 1.1.1'
 gem 'nokogiri', '~> 1.5.10'
 gem 'therubyracer', platforms: :ruby, require: linux_only('therubyracer')
-gem 'aweplug', github: 'awestruct/aweplug', ref: 'f4dd939f5a17d1946cebe5339411c411a8f483e3'
-# gem 'aweplug', github: 'Dantheman720/aweplug', :branch => 'DEVELOPER-1241'
+# gem 'aweplug', github: 'awestruct/aweplug', ref: 'f4dd939f5a17d1946cebe5339411c411a8f483e3'
+gem 'aweplug', github: 'Dantheman720/aweplug', ref: '4f0472883ebb8c6064546760c946b367336475fd'
```
Comment (paulrobinson) [inline Gemfile]: Aha, I see it's now in https://github.com/awestruct/aweplug/commits/jboss_org. Can you update this file to point to that please.

File: _config/site.yml [hunk 2]
```diff
@@ -68,10 +68,11 @@ profiles:
...
     <<: *profile
-    dcp_base_protocol_relative_url: //dcpbeta-searchisko.rhcloud.com/
-    dcp_base_url: http://dcpbeta-searchisko.rhcloud.com/
-    dcp2_base_url: http://dcp2-searchisko.rhcloud.com/
-    dcp2_base_protocol_relative_url: //dcp2-searchisko.rhcloud.com/
+    dcp_base_protocol_relative_url: //dcp.jboss.org/
+    dcp_base_url: http://dcp.jboss.org/
+    dcp2_base_protocol_relative_url: //dcp2.jboss.org/
+    dcp2_base_url: http://dcp2.jboss.org/
+    push_to_searchisko: false
```
Comment (paulrobinson) [inline _config/site.yml]: What's the reasoning here? I see you are now pointing the staging build at the production DCPs. Why did you need to do that? I see why disabling push_to_searchisko is right in this case, as we don't want the staging build pushing to the production DCP.

File: _eap-quickstarts [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit 71a4719e5f52f25857b51b3a4f2e781db5659f45
+Subproject commit d9281fa6c7ca7a498bdf95049c64bbbe41b989cf
```
Comment (paulrobinson) [inline _eap-quickstarts]: I agree we need to update the EAP quickstarts also. Did you intend to include it in this PR? I think it's safe to do so, but we need to be mindful of it.

**Comment by paulrobinson** [comment]
Currently this code will nuke the vimeo pages when doing the rsync. We want to leave them in place. To do this, I think you need to update: https://github.com/jboss-developer/www.jboss.org/blob/master/Rakefile#L176-L190

Ideally, you would ignore the 'video/vimeo' directory form the rsync. Thus any files deleted outside of that location would still get removed. Failing that, disabling the 'delete' option would work, but might come back to bite us in the future if we need to remove a page.

@LightGuard can you review this PR in some detail as well. We need to get this right as it could be catastrophic if we make a mistake.

I think we really need to test this on staging today, and then do production tomorrow (at the latest).

**Comment by LightGuard** [comment]
@paulrobinson the update has already happened where it does not do a delete. A test on staging is certainly worth doing. I'm pretty confident this will be fine.
